samples/graphene-hello/README.md

```diff
@@ -1,28 +1,6 @@
 # Graphene sample
-This sample shows how to run a [Graphene](https://github.com/oscarlab/graphene) application in Marblerun. In essence, you have to add the `premain` process to the Graphene manifest. `premain` will contact the Coordinator, set up the environment, and run the actual application. See the commented [hello.manifest.template](hello.manifest.template) for details.
+This sample shows how to run a [Graphene](https://github.com/oscarlab/graphene) application in Marblerun.
 
-You can build the sample as follows:
-```sh
-export GRAPHENEDIR=[PATH To Your Graphene Folder]
-make
-```
-Then get `mr_enclave` from the build output and set it as `UniqueID` in `manifest.json`.
+In essence, you have to add the `premain` process either by manually adding the Marblerun premain process as the entry point for your application (see: `manual-preload`), or by automatically injecting it as a preloaded shared library via Graphene's LD_PRELOAD feature (see: `auto-preload`).
```

Comment:
We may name them `fork` (or `spawn`) and `preload`, or `fork-method` and `preload-method`. Then describe the trade-offs (additional process that causes slow start vs argv cannot be provisioned).